frb_example/pure_dart/dart/lib/bridge_generated.dart

```diff
@@ -2460,64 +2460,64 @@ List<String> _wire2api_StringList(dynamic raw) {
 }
 
 String _wire2api_SyncReturn_String(dynamic raw) {
-  return utf8.decode(raw as Uint8List);
+  return utf8.decode(raw);
 }
 
 Uint8List _wire2api_SyncReturn_Uint8List(dynamic raw) {
-  return raw as Uint8List;
+  return raw;
 }
 
 bool _wire2api_SyncReturn_bool(dynamic raw) {
-  return uint8ListToBool(raw as Uint8List);
+  return uint8ListToBool(raw);
```

Comment:
nit: We can have `Uint8List raw` as arg, not `dynamic`

Reply:
Whoops, I think I accidentally pressed undo once😿